app/src/main/java/org/fossasia/openevent/general/event/EventViewHolder.kt

```diff
@@ -53,11 +52,14 @@ class EventViewHolder(override val containerView: View) : RecyclerView.ViewHolde
         val startsAt = EventUtils.getEventDateTime(event.startsAt, event.timezone)
         val endsAt = EventUtils.getEventDateTime(event.endsAt, event.timezone)
 
+        containerView.shareFab.scaleType = CENTER
+        containerView.favoriteFab.scaleType = CENTER
```

Comment:
How is this related with the issue?

Reply:
Oops. It's not. I just forgot to revert this